pyomo/repn/plugins/standard_form.py

```diff
@@ -372,7 +384,30 @@ def write(self, model):
                     f"model contains a trivially infeasible constraint, '{con.name}'"
                 )
 
-            if slack_form:
+            if mixed_form:
```

Comment:
Is it more likely that a user will want `mixed_form` over `slack_form`?

Reply:
*Users* will probably do the regular form first, and then maybe the slack form.  However, I am working on some (more efficient) solver interfaces (a rewrite of `gurobi_direct`)  that will use the `mixed_form`.